Assets/Scripts/Buddy/BuddyShaper.cs

```diff
@@ -10,27 +11,58 @@ public class BuddyShaper : MonoBehaviour
 	float legHeightOffsetMod = 0.0006f;
 
 	[SerializeField] MinMaxF exclusiveBlendRange = new MinMaxF( 50f, 100f );
-
 	SkinnedMeshRenderer skinnedMeshRend = null;
 
+	// Scaling
+	[SerializeField] MinMaxF heightScaleRange = new MinMaxF( 0.8f, 1.2f );
+
+	// Coloring
+	[SerializeField] MinMaxF colorOffsetRange = new MinMaxF( 0f, 0.15f );
+	[SerializeField] Color[] skinColors = null;
+
 	void Awake()
 	{
 		skinnedMeshRend = GetComponentInChildren<SkinnedMeshRenderer>();
-		Debug.Log( skinnedMeshRend.gameObject.name );
 
+		AdjustBlendShapes();
+		AdjustSize();
+		AdjustColor();
+	}
+
+	void AdjustBlendShapes()
+	{
 		foreach( int i in overlapBlendIndices )
 		{
 			float setWeight = Random.Range( 0f, 100f );
 			skinnedMeshRend.SetBlendShapeWeight( i, setWeight );
-
+			
 			if( i == legIndex )
 			{
 				transform.position += Vector3.up * setWeight * legHeightOffsetMod;
 			}
 		}
-
+		
 		int exclusiveBlendIndex = Random.Range( 0, exclusiveBlendIndices.Length );
 		skinnedMeshRend.SetBlendShapeWeight( exclusiveBlendIndices[exclusiveBlendIndex], 
-		                                     Random.Range( exclusiveBlendRange.min, exclusiveBlendRange.max ) );
+		                                    Random.Range( exclusiveBlendRange.min, exclusiveBlendRange.max ) );
```

Comment:
Add a test to make sure `exclusiveBlendIndicies` isn't empty or we could get an exception.